machine/accton/accton_wedge100s_32x/rootconf/sysroot-lib-onie/init-platform

```diff
@@ -0,0 +1,36 @@
+# This script fragment is sourced by /etc/init.d/init-arch.sh,
+# which in turn executes the init_platform_pre_arch() and
+# init_platform_post_arch() functions.
+
+init_platform_post_arch()
+{
+	#
+	# Wait for the EEPROM to come up.  Or more specifically,
+	# wait for the cp2112 USB-to-I2C bridge to come up.
+	# USB device initialization can happen asynchronously
+	# and late.  We want the EEPROM to be available before
+	# the rest of user-level initialization.
+	#
+	sleep 1.5
+	i=100
+	while true; do
+		if [ -r /sys/bus/i2c/devices/i2c-0/new_device ] ; then
+			echo pca9548 0x74 > /sys/bus/i2c/devices/i2c-0/new_device
+			echo 24c64 0x50 > /sys/bus/i2c/devices/i2c-7/new_device
```

Comment:
do you need to check if `/sys/bus/i2c/devices/i2c-7/new_device` exists before trying to write to it?